src/server/routes/actions.ts

```diff
@@ -53,9 +50,10 @@ const addressMiddleware = (
 
 type PartialDrip<T extends FaucetRequestType | BotRequestType> = Partial<T> & Pick<T, "address">;
 
-const dripRequestHandler = new DripRequestHandler(actions, storage, recaptchaService);
-
 router.post<unknown, DripResponse, PartialDrip<FaucetRequestType>>("/drip/web", addressMiddleware, async (req, res) => {
+  if (!config.Get("EXTERNAL_ACCESS")) {
```

Comment:
why do we need this still?

Reply:
Generally I try to have as little as possible behaviour changes when refactoring.

So here I was trying to preserve the current behaviour (`EXTERNAL_ACCESS is not set => endpoint is not working`).

But that was my only reason and I don't insist on it - the change to remove it would be small.